### PR TITLE
Fix producer

### PR DESCRIPTION
**Pull request by marinakr**

### CODE REVIEW

File: apps/core/priv/repo/migrations/20181108165546_new_persons_indexes.exs [hunk 1]
```diff
@@ -25,5 +25,9 @@ defmodule Core.Repo.Migrations.TaxIdIndex do
     execute("""
     create index if not exists persons_updated_at_index on persons(updated_at);
     """)
+
+    execute("""
+    create unique index if not exists merge_candidates_master_person_person_index on merge_candidates (master_person_id, person_id);
+    """)
```
Comment (PavelVesnin) [inline apps/core/priv/repo/migrations/20181108165546_new_persons_indexes.exs]: Migration not deployed on prod yet?

File: apps/deduplication/lib/deduplication/v2/model.ex [hunk 2]
```diff
@@ -53,8 +54,26 @@ defmodule Deduplication.V2.Model do
     end
   end
 
-  def lock_persons_on_verify(person_ids) do
-    {_, nil} = Repo.insert_all(VerifyingIds, Enum.map(person_ids, &%{id: &1}))
+  def set_current_verified_ts(updated_at) do
+    query =
+      from(p in VerifiedTs,
+        where: p.id == ^0,
```
Comment (PavelVesnin) [inline apps/deduplication/lib/deduplication/v2/model.ex]: What does it mean
'p.id == ^0'?

Comment (marinakr) [reply apps/deduplication/lib/deduplication/v2/model.ex]: this table has only one record with max updated at. We could use mnesia for this goals, but from deploy  perspective it stores in postgres. This fuction was moved from Match module

File: apps/core/priv/repo/migrations/20190207180500_mc_uniq_index.exs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+defmodule Core.Repo.Migrations.MergeCandidatesUniueIndex do
```
Comment (PavelVesnin) [inline apps/core/priv/repo/migrations/20190207180500_mc_uniq_index.exs]: Typo 'uniQue'

**Pull request merged**